man/systemctl.xml

```diff
@@ -2215,15 +2215,17 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
           locks. It takes one of <literal>auto</literal>, <literal>yes</literal> or
           <literal>no</literal>. Defaults to <literal>auto</literal>, which will behave like
           <literal>yes</literal> for interactive invocations (i.e. from a TTY) and <literal>no</literal> for
-          non-interactive invocations. <literal>yes</literal> lets the request respect inhibitor locks.
-          <literal>no</literal> lets the request ignore inhibitor locks.</para>
+          non-interactive invocations. <literal>yes</literal> lets the request check for inhibitor locks
+          before requesting the operation, so that a more friendly and informative error can be returned.
+          <literal>no</literal> skips the local check, but note that logind (since v256) will still refuse
```

Comment:
I don't agree with the change to neuter `-i/--check-inhibitors=no`. The way I understand the intended state is that we respect inhbitors **by default**, with a simplified logic that doesn't play any mind games or try to guess intent, but a privileged user is still allowed to override the inhibitors. Thus, I expect options such as `--check-inhibitors=no` and `--force`, which are documented to ignore inhibitors, to continue to do so.

(But `--force` shouldn't be the only way to ignore inhibitors. It has other side effects, for example shutdown of services is skipped. It should continue to work, but the "normal" way to ignore inhibitors should be the flag we have expressly for this purpose: `--check-inhibitors=no`.)

Reply:
ok, added a new flag that is sent when --check-inhibitors=no